README.md

```diff
@@ -647,7 +706,117 @@ jmeter:
+| description| description about the test | string | No |
+| spec | Pod specification | corev1.PodSpec | Yes |
+| displayTemplate | displayTemplate represents the output format for the results. | string | No |
+
+>Note: the only thing required in spec is the containers section with only your job container
+
+This check always display results in text based format. The `displayTemplate` defaults to `Passed: [[.passed]], Failed: [[.failed]]`
+
+The fields exposed for `displayTemplate` are:
+- .passed: number of test passed.
+- .failed: number of test failed.
+- .skipped: number of test skipped.
+- .error: number of test errored out.
+
+### Smb - Verify Folder Freshness
+
+This check connect to the specified samba server to check folder freshness. This check will:
+
+- Verify the age of most recently modified file against `minAge` and `maxAge`.
+- Verify the number of files present in the mount against `minCount`.
+```yaml
+smb:
+  - server: 192.168.1.9
+    username: samba
+    password: password
+    sharename: "Some Public Folder"
+    minAge: 10h
+    maxAge: 20h
+    searchPath: a/b/c
+    description: "Success SMB server"
+```
+
+User can define server in `\\server\e$\a\b\c` format where `server` is the host `e$` is the sharename and `a/b/c` represent the sub-dir inside mount location where the test will run to verify
+
+```yaml
+smb:
+   - server: '\\192.168.1.5\Some Public Folder\somedir'
+     username: samba
+     password: password
+     sharename: "sharename" #will be overwritten by 'Some Public Folder'
+     searchPath: a/b/c #will be overwritten by 'somedir'
+     minAge: 10h
+     maxAge: 100h
+     displayTemplate: 'Age: [[.age]]'
+     description: "Success SMB server"
+```
+> **Note:** when you use the above syntax the sharename and searchPath will be overwritten by the server
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| server | path to the server | string | Yes |
+| port | port on which smb is running. Defaults to 443 | int | No |
+| username | username for smb | string | Yes |
+| password | password for smb | string | Yes |
+| domain | domain for smb | string | No |
+| workstation | workstation for smb | string | No |
+| sharename | sharename for smb | string | No |
+| searchPath | sub-dir inside mount location inside which files will be checked | string | No |
+| minAge | minimum permissible file age | string | No |
+| maxAge | maximum permissible file age | string | No |
+| minCount | minimum number of files permissible | int | No |
+| description | description about the test | string | No |
+| displayTemplate | displayTemplate represents the output format for the results. | string | No |
+
+This check always display results in text based format. The `displayTemplate` defaults to `File Age: [[.age]]; File count: [[.count]]`
+
+The fields exposed for `displayTemplate` are:
+- .age: Age of the most recently modified file.
```

Comment:
Can we make casing on first letter consistent in the doc?

Reply:
Do you mean after the exposed fields?
in this case `Age`?